src/modules/mc_rate_control/RateControl/RateControl.cpp

```diff
@@ -90,7 +94,7 @@ Vector3f RateControl::update(const Vector3f &rate, const Vector3f &rate_sp, cons
 		updateIntegral(rate_error, dt);
 	}
 
-	return torque;
+	return _angular_accel_sp + torque_feedforward;
```

Comment:
@dagar This does not make sense to me. It seems like we are mixing torque and angular acceleration.

Reply:
@dagar Have a look at how it's done here:
https://github.com/PX4/Firmware/pull/13219/files#diff-13b03bafb0e0ca7eac858fab3d26b070R97

Reply:
@dagar Let's add a comment indicating that we are assuming an identity inertia matrix, then I think it will be more clear.

Reply:
👍